tools/ci/update_codeowners.py

```diff
@@ -0,0 +1,65 @@
+import argparse
+import os
+import re
+from typing import Optional
+
+from ..manifest import manifest
+from ..wpt import metadata, virtualenv
+
+here = os.path.dirname(__file__)
+wpt_root = os.path.abspath(os.path.join(here, os.pardir, os.pardir))
+
+
+def get_parser() -> argparse.ArgumentParser:
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--manifest", dest="manifest_path", help="Path to MANIFEST.json")
+    parser.add_argument("metadata_path", help="Path to wpt metadata repository")
+    return parser
+
+
+def run_update_codeowners(venv: virtualenv.Virtualenv,
+                          metadata_path: str,
+                          manifest_path: Optional[str],
+                          label_prefix: str = "interop-2025-",
```

Comment:
This won't capture carryover. Can you load https://github.com/web-platform-tests/wpt.fyi/blob/main/webapp/static/interop-data.json and get the set of labels from there?